perun-rpc/src/main/perl/getHostNumCores

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/perl
+
+use strict;
+use warnings;
+use Getopt::Long qw(:config no_ignore_case);
+use Text::ASCIITable;
+use Perun::Agent;
+use Perun::Common qw(printMessage tableToPrint getSortingFunction);
+
+sub help {
+	return qq{
+	Gets attribute determining number of cores on Host. Host name is required field.
+	---------------------------------------------------
+	Available options:
+	--hostName    | -H  host name
+	--batch       | -b  batch
+	--help        | -h  prints this help
+	};
+}
+
+my ($hostName, $batch);
+GetOptions("help|h" => sub { print help; exit 0;} ,
+	"hostName|H=s" => \$hostName,
+"batch|b" => \$batch) || die help;
+
+#options check
+unless(defined $hostName) { die "ERROR: hostName is required\n";}
+
+my $attrId=400;
+
+my $agent = Perun::Agent->new();
+
+my $attributesAgent = $agent->getAttributesAgent;
+my $facilitiesAgent = $agent->getFacilitiesAgent;
+
+my $table = Text::ASCIITable->new();
+$table->setCols('Id','Host Name', 'number of Cores','Facility');
```

Comment:
"Host name", "Number of cores" and "Facility name" would be more appropriate heading.